core/src/main/kotlin/io/specmatic/core/utilities/Utilities.kt

```diff
@@ -334,7 +334,9 @@ fun saveJsonFile(jsonString: String, path: String, fileName: String) {
 }
 
 fun readEnvVarOrProperty(envVarName: String, propertyName: String): String? {
-    return System.getenv(envVarName) ?: System.getProperty(propertyName)
```

Comment:
What's the reason for this change?

Reply:
On retrospect I don't need this change. I'll revert. I had added this because I thought if I set env var as empty string, it won't work properly unless I return null from this function. That was an incorrect assumption.

Reply:
Addressed.